tools/optimize/dx12_optimize_util.cpp

```diff
@@ -104,22 +104,23 @@ bool BypassDxrOptimization(gfxrecon::decode::Dx12ObjectScanningConsumer& resref_
 {
     bool bypass_dxr_optimization = false;
 
-    bool contains_dxr_workload = resref_consumer.ContainsDxrWorkload();
-    bool is_dxr_optimized      = resref_consumer.ContainsOptFillMem();
+    bool contains_dxr_ei_workload = resref_consumer.ContainsDxrWorkload() || resref_consumer.ContainsEiWorkload();
+    bool is_dxr_optimized         = resref_consumer.ContainsOptFillMem();
```

Comment:
The optimization is both DXR and EI, is that right?  Is there a better name for this variable and `bypass_dxr_optimization` then?  Like `is_optimized_for_address_within_buffer`?  Or something less verbose, maybe even `is_optimized` and `bypass_optimization`?

Reply:
`contains_dxr_ei_workload` could become `bypass_rv_workload`

`is_dxr_optimized` could become `is_rv_optimized`

rv = resource value

Resource values represent that type of work. Using `bypass_optimization` is a bit misleading because there's also PSO optimization step.

Reply:
Optimizer variables have been renamed from **dxr** to **resource_value**. Resource value is how GFXR internally refers to GPU VAs, GPU descriptors, shader IDs that an application can write to data and must be mapped for repaly.

The user-facing argument option for `gfxrecon-optimize` has not changed. It is `--dxr`.